scripts/install-go.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
+1 on the rename. It has grown beyond the simple install thingy it started out as.